data/odyssey_functions.csv

```diff
@@ -22224,22 +22224,22 @@ Address,Quality,Size,Name
 0x000000710046e794,U,001068,_ZN19PlayerStateGrabCeil8exeSwingEv
 0x000000710046ebc0,U,000224,_ZN19PlayerStateGrabCeil7exeJumpEv
 0x000000710046eca0,U,000036,_ZN19PlayerStateGrabCeilD0Ev
-0x000000710046ecec,U,000212,_ZN21PlayerStateGroundSpinC2EPN2al9LiveActorEPK19IUsePlayerCollisionPK11PlayerInputPK11PlayerConstP14PlayerAnimator
-0x000000710046edc0,U,000068,_ZN21PlayerStateGroundSpin6appearEv
-0x000000710046ee04,U,000576,_ZN21PlayerStateGroundSpin13exeGroundSpinEv
-0x000000710046f044,U,000036,_ZN21PlayerStateGroundSpinD0Ev
-0x000000710046f070,U,000216,_ZN15PlayerStateHackC2EPN2al9LiveActorEP16PlayerHackKeeperP19IPlayerModelChangerP14PlayerAnimatorP7HackCap
-0x000000710046f148,U,000072,_ZN15PlayerStateHack6appearEv
-0x000000710046f190,U,000284,_ZN15PlayerStateHack16prepareStartHackEPKN2al9HitSensorES3_
-0x000000710046f2ac,U,000176,_ZN15PlayerStateHack14prepareEndHackEv
-0x000000710046f35c,U,000100,_ZN15PlayerStateHack21prepareStageStartHackEv
-0x000000710046f3c0,U,000056,_ZNK15PlayerStateHack22isIgnoreUpdateColliderEv
-0x000000710046f3f8,U,000108,_ZNK15PlayerStateHack19isEnableChangeStateEv
-0x000000710046f464,U,000068,_ZNK15PlayerStateHack25isEnableModelSyncShowHideEv
-0x000000710046f4a8,U,000088,_ZNK15PlayerStateHack18isEnableCancelHackEv
-0x000000710046f500,U,001676,_ZN15PlayerStateHack11exeHackDemoEv
-0x000000710046fb8c,U,000196,_ZN15PlayerStateHack21exeHackDemoPuppetableEv
-0x000000710046fc50,U,000084,_ZN15PlayerStateHack7exeHackEv
+0x000000710046ecec,O,000212,_ZN21PlayerStateGroundSpinC2EPN2al9LiveActorEPK19IUsePlayerCollisionPK11PlayerInputPK11PlayerConstP14PlayerAnimator
+0x000000710046edc0,O,000068,_ZN21PlayerStateGroundSpin6appearEv
+0x000000710046ee04,O,000576,_ZN21PlayerStateGroundSpin13exeGroundSpinEv
+0x000000710046f044,W,000036,_ZN21PlayerStateGroundSpinD0Ev
+0x000000710046f070,W,000216,_ZN15PlayerStateHackC2EPN2al9LiveActorEP16PlayerHackKeeperP19IPlayerModelChangerP14PlayerAnimatorP7HackCap
```

Comment:
i didn't realize that was even a thing. can you explain the difference? also, was it something i was supposed to edit myself.. ? can do though

Reply:
https://github.com/open-ead/nx-decomp-tools/blob/master/viking/src/functions.rs#L23
There are different "states" for functions: Matching (O) is the best, NotDecompiled (U) the most common.
In-between, we have non-matching - major (M) and minor (m) to mark functions which are basically done, but don't match for some reason.
There is also Library (L), but not sure when this would be used.

If you don't manage to get a perfect match, you should manually edit the state to be `M` or `m`, based on how close you are.